test/tests/functional/pbs_pp369.py

```diff
@@ -36,10 +36,10 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
-from ptl.utils.pbs_testsuite import *
+from tests.functional import *
 
 
-class TestPp369(PBSTestSuite):
+class TestPp369(TestFunctional):
```

Comment:
Is changing the testsuite names in scope for this task?  The Jira issue pertains only to the directory structure.